helper_scripts/analyze.py

```diff
@@ -49,9 +49,18 @@ def analyze_kafka_dump(directory):
             writer.writerow([merchant_id_mapping[merchant_id], revenue[merchant_id], holding_cost[merchant_id],
                              order_cost[merchant_id], profit[merchant_id]])
 
-    create_inventory_graph(directory, merchant_id_mapping)
-    create_profit_per_minute_graph(directory, merchant_id_mapping)
-    create_revenue_per_minute_graph(directory, merchant_id_mapping)
+    create_chart(directory, merchant_id_mapping,
+        topic='inventory_level', value_name='level', label='Inventory Level',
+        filename='inventory_levels.png', drawstyle='steps-post')
+    create_chart(directory, merchant_id_mapping,
+        topic='profitPerMinute', value_name='profit', label='Profit per Minute', filename='profit_per_minute.png')
+    create_chart(directory, merchant_id_mapping,
+        topic='revenuePerMinute', value_name='revenue', label='Revenue per Minute', filename='revenue_per_minute.png')
+    create_chart(directory, merchant_id_mapping,
+        topic='profit', value_name='profit', label='Cumulative Profit', filename='cumulative_profit.png')
+    create_chart(directory, merchant_id_mapping,
+        topic='cumulativeRevenue', value_name='revenue', label='Cumulative Revenue', filename='cumulative_revenue.png')
```

Comment:
Please see my comment from 7th of July.

Reply:
Ping @CarstenWalther  :)

Reply:
Now the graph is skipped if a topic file is missing. This kafka topic is new. Are you on the newest version of the `analytics` subrepo.

Reply:
Yes, I have been using the latest analytics master commit Anyhow ... merging now.